ui/src/pages/namespace/Explorer/Tree/components/modals/CreateNew/Workflow/index.tsx

```diff
@@ -89,7 +90,7 @@ const NewWorkflow = ({
     },
   });
 
-  const { mutate: createWorkflow, isLoading } = useCreateWorkflow({
+  const { mutate: createFile, isLoading } = useCreateFile({
     onSuccess: (data) => {
```

Comment:
I noticed that we now have a lot of `data.data` in our codebase. We could easily fix most of them by renaming some vars/props. Like e.g. in this case we could change `onSuccess: (data) => {` to `onSuccess: (file) => {`. I'm just not sure if it's that important to handle that right now. Maybe you could search for `data.data` and fix the low-hanging fruits that you find?

Reply:
As discussed, I am skipping this. Our use of `data` as a first level object is quite consistent, so the inconsistency arising out of renaming some of these objects may be worse than encountering `data.data` here and there.